charts/eks-anywhere-packages/values.yaml

```diff
@@ -85,4 +85,14 @@ controller:
       memory: 50Mi
     limits:
       cpu: 500m
-      memory: 300Mi
\ No newline at end of file
+      memory: 300Mi
+# Additional Variables to configure cronjob
+cronjob:
+  name: ecr-refresher
+  # -- ECR refresher repository name.
```

Comment:
Won't we be targeting this to another repository?

Reply:
Updated to different repository